File: fc-chat/demo/part3/ch4/message-system-add-push/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/FetchChannelsResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import java.util.List;
import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.dto.domain.Channel;

public class FetchChannelsResponse extends BaseMessage {

  private final List<Channel> channels;

  public FetchChannelsResponse(List<Channel> channels) {
    super(MessageType.FETCH_CHANNELS_RESPONSE);
    this.channels = channels;
  }

  public List<Channel> getChannels() {
    return channels;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/FetchChannelsResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.constant.MessageType
import com.yscorp.withpush.messagesystem.dto.domain.Channel


class FetchChannelsResponse(val channels: List<Channel>) : BaseMessage(MessageType.FETCH_CHANNELS_RESPONSE) {
}
